Firestore/Source/Public/FIRDocumentReference.h

```diff
@@ -195,6 +196,11 @@ NS_SWIFT_NAME(DocumentReference)
 - (void)getDocumentWithCompletion:(FIRDocumentSnapshotBlock)completion
     NS_SWIFT_NAME(getDocument(completion:));
 
+// clang-format off
+- (void)getDocumentWithOptions:(FIRGetOptions *)options completion:(FIRDocumentSnapshotBlock)completion
+    NS_SWIFT_NAME(getDocument(options:completion:));
```

Comment:
Please add a swift-language test that verifies this looks correct from swift.

Reply:
Done.